versions.go

```diff
@@ -7,15 +7,29 @@ import (
 	"github.com/google/go-github/github"
 )
 
-// determineVersionFromTag converts git tags v1.2.3 into semver 1.2.3 values
-func determineVersionFromTag(tag string) string {
-	re := regexp.MustCompile("v?([^v].*)")
-	matches := re.FindStringSubmatch(tag)
+var defaultTagFilter = "v?([^v].*)"
```

Comment:
We may want to make the default regex anchored at the beginning, as that's what originally broke with #54.